dev/com.ibm.websphere.appserver.features/visibility/public/faces-3.0/io.openliberty.faces-3.0.feature

```diff
@@ -40,7 +40,7 @@ Subsystem-Name: Jakarta Server Faces 3.0
  io.openliberty.jakarta.validation-3.0, \
  io.openliberty.jakarta.faces-3.0, \
  io.openliberty.pages-3.0, \
- io.openliberty.jakarta.jaxb-3.0; \
+ io.openliberty.jakarta.xmlBinding-3.0; \
```

Comment:
I think `'; \` should be `, \`

Reply:
agreed - though that was there before...  I'll change it.